src/commands/flinkStatements.ts

```diff
@@ -48,43 +48,29 @@ const MAX_WAIT_TIME_MS = 60_000;
  */
 async function waitForStatementRunning(
```

Comment:
The respelled version is much easier to [read as a whole,](https://github.com/confluentinc/vscode/blob/jlr/1677-fetch-individual-flink-statement/src/commands/flinkStatements.ts#L49-L74) than this diff.